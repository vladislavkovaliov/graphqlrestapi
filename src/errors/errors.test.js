const { AUTHENTICATION_ERROR, UNAUTHOTIZED } = require('./errors');

describe('[errors.js]', () => {
  test('should authentication error be correctly', () => {
    expect(AUTHENTICATION_ERROR).toMatchSnapshot();
  });

  test('should unauthotized be correctly', () => {
    expect(UNAUTHOTIZED).toMatchSnapshot();    
  });
});