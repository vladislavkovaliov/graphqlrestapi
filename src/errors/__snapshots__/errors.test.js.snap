// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[errors.js] should authentication error be correctly 1`] = `
Object {
  "msg": "Authentication failed. Wrong password.",
  "statusCode": 401,
  "success": false,
}
`;

exports[`[errors.js] should unauthotized be correctly 1`] = `
Object {
  "msg": "Unauthorized.",
  "statusCode": 403,
  "success": false,
}
`;
